test/generators/install_generator_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+require_relative File.expand_path("../../../lib/generators/accessly/install/install_generator", __FILE__)
+
+class Accessly::InstallGeneratorTest < ::Rails::Generators::TestCase
+  tests Accessly::Generators::InstallGenerator
+  destination File.expand_path("../../tmp", File.dirname(__FILE__))
+
+  setup do
+    run_generator
+  end
+
+  Minitest.after_run do
+    FileUtils.rm_rf(destination_root)
+  end
+
+  test "generates migrations" do
+    migration1 = migration_file_name("db/migrate/create_permitted_actions.rb")
+    migration2 = migration_file_name("db/migrate/create_permitted_action_on_objects")
+    bad_migration = migration_file_name("db/migrate/accessly_not_exist")
+
+    migration1.wont_be_nil
+    migration2.wont_be_nil
```

Comment:
@armilam a few PRs ago I introduced `assert_nil` on the deprecated recommendation to use that over `must_equal nil`

`wont_be_nil` and `must_be_nil` can provide the consistency you were looking for and so I plan to open a different pr to change those assertions